dev-docs/bidders/exco.md

```diff
@@ -21,18 +21,19 @@ fpd_supported: false
 ortb_blocking_supported: false
 multiformat_supported: will-bid-on-one
 pbjs: true
+pbs: true
 sidebarType: 1
 ---
 
 ### Bid Params
 
 {: .table .table-bordered .table-striped }
 
-| Name       | Scope    | Description                                                                               | Example                      | Type     |
-|------------|----------|-------------------------------------------------------------------------------------------|------------------------------|----------|
-| `cId`      | required | The connection ID from Exco.                                                          | `'562524b21b1c1f08117fc7f9'` | `string` |
```

Comment:
Sorry @pb-pete - we can't let you pull the documentation rug out from under any publishers that were using `exco` before this change.

Not clear to me that the PBJS team should have let you merge such a drastic change in parameters outside of a major release, but for documentation, I'm going to ask that you leave cId and pId in this table flagged as deprecated. e.g. "Deprecated with Prebid.js 9.33. Please use accountId and publisherId instead."